java/colormap-renderer/README.md

```diff
@@ -28,13 +35,18 @@ Simply start the app and allow read permissions.
 6. Create the ArcGIS/samples directory, `mkdir ArcGIS/samples/raster`.
```

Comment:
i don't need to run this mkdir command with adb, I can push a file to a dir that doesn't exist and it will create the dir. maybe this is a problem on other platforms or oder version of adb. just an observation :)

Reply:
I would say probably best to leave it, as telling people to push to a nonexistant directory might be confusing ("I can't do this, I don't have this directory, should I just make it? Should it already exist?")

Reply:
I agree. I have noticed that many other (I guess newer) samples do not instruct the user to mkdir, but just assume the dir is there. But I think that is maybe risky and that this is maybe better.